apps/sms/views/inbox/js/startup.js

```diff
@@ -39,6 +40,7 @@
       ConversationClient.init(App.instanceId);
       MessageManager.init();
       Navigation.init();
+      Drafts.init();
```

Comment:
__optional nit:__ not critical right now, but maybe we can also add this call to coversation/js/startup.js in case we add more code to ```Drafts.init``` that will be needed by Conversation, but it's optional